lib/market_bot/play/app.rb

```diff
@@ -118,7 +118,12 @@ def self.parse(html, _opts = {})
           end.compact.uniq
         end
 
-        h2_more = doc.at_css("h2:contains(\"#{result[:developer]}\")")
+        if result[:developer].include? "'"              # Add the wrapper to avoid quote's parsing issues
+          css_selector = "h2:contains(\"#{result[:developer]}\")"
```

Comment:
@Kligerr you will have a problem here if the text includes both ' and " - you need to somehow escape it

Reply:
@borisspektor Fixed

Reply:
@borisspektor I played around it but didn't succeed to get a shorter (but working 😛) solution